src/databricks/labs/remorph/deployment/job.py

```diff
@@ -108,7 +108,9 @@ def _recon_job_settings(
 
     def _job_recon_task(self, jobs_task: Task, recon_config: ReconcileConfig) -> Task:
         libraries = [
-            compute.Library(pypi=compute.PythonPyPiLibrary("databricks-labs-remorph")),
+            compute.Library(
+                whl=f"/Workspace/{self._installation.install_folder()}/wheels/databricks_labs_remorph-{self._product_info.version()}-py3-none-any.whl"
```

Comment:
use the result of `self._wheels.upload_to_wsfs()`, otherwise we won't be able to write integration tests over current branch. e.g. `self._product_info.version()` is `0.5.1+920240404234234`, where `9` is the number of commits from the last tagged release and the rest is a timestamp. see https://github.com/databrickslabs/ucx/blob/main/src/databricks/labs/ucx/installer/workflows.py#L609

Reply:
Updated the code, please review @nfx